packages/react-dom/src/__tests__/ReactDOMFizzServer-test.js

```diff
@@ -135,14 +168,15 @@ describe('ReactDOMFizzServer', () => {
     while (fakeBody.firstChild) {
       const node = fakeBody.firstChild;
       if (
-        node.nodeName === 'SCRIPT' &&
+        (node.nodeName === 'SCRIPT' || node.nodeName === 'script') &&
         (CSPnonce === null || node.getAttribute('nonce') === CSPnonce)
       ) {
-        const script = document.createElement('script');
+        const script = document.createElement('SCRIPT');
         script.textContent = node.textContent;
         fakeBody.removeChild(node);
         parent.appendChild(script);
       } else {
+        replaceScripts(node);
```

Comment:
What's this doing? It looks like it's just doing the same thing as above but recursively?

We can kind of assume that the script tags are always inserted in the body since that what we do.